packages/terra-functional-testing/lib/test-runner/test-runner.js

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs');
+const path = require('path');
+const Launcher = require('@wdio/cli').default;
+
+class TestRunner {
+  /**
+   * Resolves the configuration file path.
+   * If no path is provided the current working directory will be searched. Falls back to the default configuration file.
+   * @param {string} configPath - The path to the test runner configuration file.
+   * @returns {string} A resolved path to the test runner configuration file.
+   */
+  static configPath(configPath) {
+    if (configPath) {
+      return path.resolve(configPath);
+    }
+
+    const relativeConfigPath = path.resolve(process.cwd(), 'wdio.conf.js');
+
+    if (fs.existsSync(relativeConfigPath)) {
+      return relativeConfigPath;
+    }
+
+    return path.resolve(__dirname, './wdio.conf.js');
+  }
+
+  /**
+   * Starts the test runner.
+   * @param {Object} options - The test run options.
+   * @returns {Promise} A promise that resolves with the test run exit code.
+   */
+  static async run(options) {
+    try {
```

Comment:
Is it possible to use promise syntax here?

Reply:
Promises and async/awaits are interchangeable under most circumstances. I chose async/await here for the explicit and newer syntax.